index.html

```diff
@@ -165,11 +165,11 @@ <h2>About Myself</h2>
                         <a href="#"><img src="./image/union.svg" alt="Union" /></a> 
                     </div>
                       <ul class="language-list">
-                        <li><a href="#" class="language-icon"><img src="./image/js.png" alt="" /></a></li>
-                        <li><a href="#" class="language-icon"><img src="./image/html.png" alt="" /></a></li>
-                        <li><a href="#" class="language-icon"><img src="./image/css.png" alt="CSS" /></a></li>
-                        <li><a href="#" class="framework-icon"><img src="./image/frameworks.png" alt="frameworks"/></a></li>
-                        <li><a href="#" class="framework-icon"><img src="./image/skills.png" alt="Skills" /></a></li>
+                        <li><a href="#" class="language-icon"><img src="./image/js.png" alt="Javascript Icon" /></a></li>
+                        <li><a href="#" class="language-icon"><img src="./image/html.png" alt="Html Icon " /></a></li>
+                        <li><a href="#" class="language-icon"><img src="./image/css.png" alt="Css Icon" /></a></li>
+                        <li><a href="#" class="framework-icon"><img src="./image/frameworks.png" alt="Frameworks Icon"/></a></li>
+                        <li><a href="#" class="framework-icon"><img src="./image/skills.png" alt="Skills Icon" /></a></li>

```

Comment:
- [ ] Kindly be noticed that there is an extra space at the bottom of the page please remove it also you should make the contact form as the Figma design.

![Screenshot from 2022-07-19 21-44-10](https://user-images.githubusercontent.com/33752735/179836434-211c9014-be2c-43ea-81c6-c272d89c7abd.png)

Your Design | Figma Design
---------------- | --------------------
![Screenshot from 2022-07-19 21-49-17](https://user-images.githubusercontent.com/33752735/179836368-82abc6ab-ae5b-4bea-a0a5-d07bb84ec3b0.png) | ![Screenshot from 2022-07-19 21-45-00](https://user-images.githubusercontent.com/33752735/179836394-fa6d6d72-0a62-4afc-bca9-8daca8abdf94.png)

Reply:
@ShahierNashaat thank you for your code review feedback. Just check my page margins(Bottom-margin) and it looks just fine as given in the Figma design. Can you please verify once again if indeed there is no margin at the bottom?